### DISCUSSION TITLE
ERGO EIP27 compatibility

### DISCUSSION THREAD

**Discussion by ei8ht187** [coinfoundry/miningcore#1261] [Q&A]
hello, 
is miningcore already compatible with ERGOs EIP27 change? 
https://ergoplatform.org/en/blog/Ergo%E2%80%99s-EIP-27-Rollout/

**Comment by oliverw** [comment]
Working on it.

**Reply by ei8ht187** [reply]
Any ETA? EIP27 will go live in about 24h

**Reply by oliverw** [reply]
Still waiting for a reply to some of my questions from the Ergo Devs 😞

**Reply by MiningCryptoLive** [reply]
There shouldn't be anything that needs to be changed in miningcore. The changes to enable the EIP are to be made in the ergo nodes conf file.
https://github.com/ergoplatform/ergo/releases/tag/v4.0.31

**Reply by ei8ht187** [reply]
> There shouldn't be anything that needs to be changed in miningcore. The changes to enable the EIP are to be made in the ergo nodes conf file. https://github.com/ergoplatform/ergo/releases/tag/v4.0.31

Yes, the configuration is all set. I just hope that MC get's the lower paid amount and calculates the payouts correctly. Code for it looking good so far, it is no hardcoded amount for the payout. My guess is also, it will be working.

**Reply by oliverw** [reply]
My tests on testnet were successful.

**Comment by ei8ht187** [comment]
Please check issue #1280
